README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantics give the bones of the website meaning, logic, and order. Gives the website a basic flow. 
```

Comment:
A little nitpicky, but HTML I would say gives the website logic and order. Semantics give those "bones", the logic and order, a "why": what is the topic of this page, how is the content organized, etc.